File: SecondYear/ApoorvaSri123_ApoorvaSrivastava_2226it1116_2/Week_1/user_input_output.c
#include <stdio.h>
#include <conio.h>

int main() {
    char ch;

    printf("Enter a character: ");
    scanf(" %c", &ch);

    if (ch>='A'&& ch<='Z') {
        printf("1\n");
    } else if (ch>='a'&& ch<='z') {
        printf("0\n");
    } else{
        printf("-1\n");
    }
return 0;
    
}

File: SecondYear/riababbar4_RiaBabbar_2226en1042_2/Week_1/user input_output.py
def check_char_type(char):
    if 'A' <= char <= 'Z':
        return 1
    elif 'a' <= char <= 'z':
        return 0
    else:
        return -1

# Taking input from the user
user_input = input("Enter a character: ")

# Checking and printing the result
result = check_char_type(user_input)
print(result)